pkg/machines/components/create-vm-dialog/cloudInitOptionsRow.jsx

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Please squash the "move cloud init related code to seperate file" commit (also, this should look different after factorizing)

Reply:
I decided to do any code move in follow up, no need to be part of this PR.